frontend/rust-lib/event-integration/tests/folder/local_test/search_test.rs

```diff
@@ -0,0 +1,46 @@
+use crate::folder::local_test::script::{create_view, FolderTest};
+use collab_folder::ViewLayout;
+use std::time::Duration;
+
+#[tokio::test]
```

Comment:
How about adding these tests:

### 1. Test with Large Folder
**Goal**: Make sure the app works fine with a big folder.
- Create a big folder with lots of files.
- Use the app to open and work with this folder.
- **Expected**: The app should not crash or slow down too much.

### 2. Test Deleting a View
**Goal**: Check if a deleted view disappears from search results.
- Delete a view named "View A".
- Search for "View A".
- **Expected**: "View A" should not show up in search results.
- Then, search in the trash if we supports it.
- **Expected**: "View A" should be found in the trash.

### 3. Test Renaming and Searching a View
**Goal**: Ensure the app can find a view by its new name after renaming.
- Rename "View B" to "View C".
- Search for "View B".
- **Expected**: No results for "View B".
- Search for "View C".
- **Expected**: "View C" shows up in search results.